docs/v3.x/getting-started/react.md

```diff
@@ -0,0 +1,186 @@
+```
+
+:::
+
+::: tab npx
+
+```bash
+npx create-react-app react-app
+```
+
+:::
+
+::::
+
+### Use an HTTP client
+
+Many HTTP clients are available but in this documentation we'll use [Axios](https://github.com/axios/axios) and [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
+
+
+:::: tabs
+
+::: tab axios
+
+```bash
+yarn add axios
+```
+
+:::
+
+::: tab fetch
+
+No installation needed
+
+:::
+
+::::
+
+### GET Request your collection type
+
+Execute a GET request on the `restaurant` Collection Type. 
+
+:::: tabs
+
+::: tab axios
+
+```js
+axios.get('http://localhost:1337/restaurants')
+  .then(response => {
+    // handle success
+    console.log(response);
+  })
+  .catch(error => {
+    // handle error
+    console.log(error);
+  })
+  .then(function () {
+    // always executed
+  });
+
+// response = [{"id":1,"name":"Biscotte Restaurant","description":"Welcome to Biscotte restaurant! Restaurant Biscotte offers a cuisine based on fresh, quality products, often local, organic when possible, and always produced by passionate producers.","created_by":{"id":1,"firstname":"Paul","lastname":"Bocuse","username":null},"updated_by":{"id":1,"firstname":"Paul","lastname":"Bocuse","username":null},"created_at":"2020-07-31T11:37:16.964Z","updated_at":"2020-07-31T11:37:16.975Z","categories":[{"id":2,"name":"French Food","created_by":1,"updated_by":1,"created_at":"2020-07-31T11:36:23.164Z","updated_at":"2020-07-31T11:36:23.172Z"}]}]
+```
+
+:::
+
+::: tab fetch
+
+```js
+fetch("http://localhost:1337/restaurants")
+  .then(response => response.json())
+  .then(data => console.log(data));
+
+// data = [{"id":1,"name":"Biscotte Restaurant","description":"Welcome to Biscotte restaurant! Restaurant Biscotte offers a cuisine based on fresh, quality products, often local, organic when possible, and always produced by passionate producers.","created_by":{"id":1,"firstname":"Paul","lastname":"Bocuse","username":null},"updated_by":{"id":1,"firstname":"Paul","lastname":"Bocuse","username":null},"created_at":"2020-07-31T11:37:16.964Z","updated_at":"2020-07-31T11:37:16.975Z","categories":[{"id":2,"name":"French Food","created_by":1,"updated_by":1,"created_at":"2020-07-31T11:36:23.164Z","updated_at":"2020-07-31T11:36:23.172Z"}]}]
+```
+
+:::
+
+::::
+
+### Examples
+
+:::: tabs
+
+::: tab axios
+
+`./src/App.js`
+
+```js
+import React from 'react';
+import axios from "axios"
+
+class App extends React.Component {
+
+  state = {
+    restaurants: []
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:1337/restaurants')
+      .then(response => {
+        const restaurants = response.data;
+        this.setState({ restaurants });
+      })
+      .catch(function (error) {
+        console.log(error);
+      })
+      .then(function () {
+      });
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <ul>
+          { this.state.restaurants.map(restaurant => <li key={restaurant.id}>{restaurant.name}</li>)}
```

Comment:
When using classes instead of functional components it is best to avoid using functions in the render method:

```js
renderRestaurant = restaurant => <li key={restaurant.id}>{restaurant.name}</li>

render() {
  // ....

  return (
    <div className="App">
      <ul>{restaurants.map(renderRestaurant)}</ul>
    </div>
);
```